LibreNMS/Authentication/LdapAuthorizer.php

```diff
@@ -161,6 +161,7 @@ public function getUserlist()
                                 'realname' => $realname,
                                 'user_id' => $user_id,
                                 'email' => $email,
+                                'level' => $this->getUserlevel($username),
```

Comment:
This is going to make getUserlist() even slower. Can you please add it on the getUser() function and make mark a TODO here.

Reply:
I am not sure why a TODO should be added to getUserlist (and what to write as a description of the TODO).

Maybe this fix should be added to #7718?